README.md

```diff
@@ -120,6 +120,10 @@ This is called with an object with the properties listed below:
 | `selectItemAtIndex`     | `function(index: number)`  | selects the item at the given index                                                                              |
 | `selectHighlightedItem` | `function()`               | selects the item that is currently highlighted                                                                   |
 
+### Autocomplete.Button
```

Comment:
There aren't any props 😁 we pull `disabled` off, but I think it's fine since that's a normal prop for button.